cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -1172,7 +1181,7 @@ func newServer(conf rootConfig) (*Server, error) {
 
 func main() {
 	liblog := sglog.Init(sglog.Resource{
-		Name:       "zoekt-indexserver",
+		Name:       "zoekt-sourcegraph-indexserver",
```

Comment:
I know the binary name is `zoekt-sourcegraph-indexserver`, but everywhere else we refer to the container as `zoekt-indexserver`. @bobheadxi any downside to this being a bit more of a verbose name?

Reply:
I would advocate that this align with how we reference the service elsewhere, which I think is `zoekt-indexserver`